Makefile

```diff
@@ -14,9 +14,11 @@ protolint:
 gomodtidy:
 	go mod tidy -v
 
-generate:
+generate_code:
```

Comment:
[for reviewer] this is not required for this PR, and there is probably a better way to deal with this.

Reply:
This seems fine to me, thanks